querydsl-apt/src/test/apt/com/querydsl/ExampleEntity2.java

```diff
@@ -0,0 +1,10 @@
+package com.querydsl;
+
+import com.querydsl.core.annotations.QueryEntity;
+
+@QueryEntity
+public class ExampleEntity2 {
```

Comment:
I forgot one detail of this test case, it should extend an external `@QueryEntity` so that it inherits a `NumberPath` that is normally not assignable to a QInteger.
```java
private final QInteger integerProp = _super.integerProp; // doesn't work.
```